src/github.com/getlantern/flashlight/settings/settings.go

```diff
@@ -110,5 +112,8 @@ func read() {
 			}
 			return nil
 		})
+		if err != nil {
+			log.Debugf("Unable to update settings: %v", err)
```

Comment:
I think this should probably be logged as an error too, although I would imagine settings logs it internally most likely.